### PR TITLE
Update Release documentation.

### PR DESCRIPTION
**Pull request by gtrevi**
## Description

Updated the list of files to be released.

## Testing

v0.11.0 release process.

## Documentation

`ReleaseProcess.md`

## Installation

n.a.

### CODE REVIEW

File: docs/ReleaseProcess.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ eBPF for Windows, and how to service it later.
 
 ## Creating a new release
 
-1. Create a working branch from "`main`" on your private repo fork, and check it out.
+1. Create a working branch from "`main`" on your private repo fork, and check it out (e.g.: `<user>/release-X-Y-Z`).
```
Comment (dthaler) [inline docs/ReleaseProcess.md:10]:
```suggestion
1. Create a working branch from "`main`" on your private repo fork, and check it out (e.g., `<user>/release-X-Y-Z`).
```

File: docs/ReleaseProcess.md [hunk 2]
```diff
@@ -127,10 +127,15 @@ As a result of creating new release or servicing an existing one, the following
 1. Click on the "`Choose a tag`" combo box and select the tag with new "`vX.Y.Z`" version number, as created earlier.
 1. Fill in the release title as "`vX.Y.Z`" (replace "`X.Y.Z`" with the version number being released).
 1. Manually enter release notes or click "`Generate release notes`" and then edit as desired.
-1. Attach the `.msi` and the (non-redist) `.nupkg` files by dropping them in the "`Attach binaries by dropping them here or selecting them.`" area.
+1. Attach the `.msi`, the (non-redist) `.nupkg` and the `Build-x64-native-only-Release.X.Y.Z.zip` build (from the CI/CD artifacts), by dropping them in the "`Attach binaries by dropping them here or selecting them.`" area. For example, the file list for `v0.11.0` should be:
```
Comment (dthaler) [inline docs/ReleaseProcess.md:130]:
```suggestion
1. Attach the `.msi`, the (non-redist) `.nupkg`, and the `Build-x64-native-only-Release.X.Y.Z.zip` build (from the CI/CD artifacts), by dropping them in the "`Attach binaries by dropping them here or selecting them.`" area. For example, the file list for `v0.11.0` should be:
```

**Review by dthaler — APPROVED** [review]
Ok modulo two punctuation nits

File: docs/ReleaseProcess.md (hunk 1 above)
Comment (gtrevi) [reply docs/ReleaseProcess.md:10]: updated

File: docs/ReleaseProcess.md (hunk 2 above)
Comment (gtrevi) [reply docs/ReleaseProcess.md:130]: updated

**Review by shpalani — APPROVED** [review]

**Review by dthaler — APPROVED** [review]

**Pull request merged**